docs/configuration/shortener.rst

```diff
@@ -0,0 +1,28 @@
+.. vale off
+
+Shortener service
+#########
+
+.. attention::
+
+    Mautic 5 introduces a new shortening feature, replacing the previous legacy shortening system found in the settings. If you utilized the shortening service in Mautic 4, upon upgrading to Mautic 5, please follow the steps below to reconfigure it.
+
+The new shortener service allows developers to create plugins for any shortener service, with any type of authorization. By default, Mautic does not provide a shortener service plugin; you must install it either from the marketplace or manually.
+
+
+Example of setup Bitly plugin
+=================
+
+1. Install the Bitly bundle from the marketplace or using Composer:
+
+.. code-block:: bash
+
+    composer require webmecanik/mautic-bitly-bundle
+
+2. Obtain an access key from https://app.bitly.com/settings/api and set up/enable the Bitly plugin.
```

Comment:
```suggestion
2. Obtain an access key from :xref:`Bitly API settings` and set up/enable the Bitly plugin.
```
If you need to use a link, please cd into the docs directory and use the command `make link` - follow the steps and commit the link file that is created. This hooks in with the automatic link checking linter and avoids duplication! I will commit the file to your PR to save you having to do it this time.